--- external/CMakeLists.txt.orig
+++ external/CMakeLists.txt
@@ -46,8 +46,8 @@	
 add_subdirectory(./solanaceae_ircclient)
 set(SOLANACEAE_CRDTNOTES_BUILD_PLUGINS ON CACHE BOOL "")
 add_subdirectory(./solanaceae_crdtnotes)

-set(SOLANACEAE_LLAMA-CPP-WEB_BUILD_PLUGINS ON CACHE BOOL "")
-add_subdirectory(./solanaceae_llama-cpp-web)
+#set(SOLANACEAE_LLAMA-CPP-WEB_BUILD_PLUGINS ON CACHE BOOL "")
+#add_subdirectory(./solanaceae_llama-cpp-web)

 set(SOLANACEAE_VCG_BUILD_PLUGINS ON CACHE BOOL "")
 set(SOLANACEAE_VCG_BUILD_TESTING ${SOLANACEAE_ECOSYSTEM_BUILD_TESTING} CACHE BOOL "")
